libs/datasets/model_output_schema.py

```diff
@@ -0,0 +1,25 @@
+MODEL_OUTPUT_SCHEMA = [
+    "day_num",
+    # ^ for index column
+    "date",
+    "a",
+    "b",
+    "c",
+    "d",
+    "e",
+    "f",
+    "g",
+    "all_hospitalized",
+    "all_infected",
+    "dead",
+    "beds",
+    "i",
+    "j",
+    "k",
+    "l",
+    "population",
+    "m",
+    "n",
+]
+
+MODEL_OUTPUT_SCHEMA_EXCLUDED_COLUMNS = ["a","b","c","d","e","f","g","i","j","k","l","m","n"]
```

Comment:
What are these excluded from?

Reply:
when we pull in the data we ignore them. they should be named lol